vmm/src/cpu.rs

```diff
@@ -1129,17 +1129,21 @@ impl CpuManager {
     }
 
     pub fn resize(&mut self, desired_vcpus: u8) -> Result<bool> {
-        if !self.dynamic {
-            return Err(Error::ResizingNotSupported);
-        }
-
         match desired_vcpus.cmp(&self.present_vcpus()) {
             cmp::Ordering::Greater => {
+                if !self.dynamic {
```

Comment:
You could put an equality check at the top of the function and return `Ok(false)` there instead (would be less code.)